index.html

```diff
@@ -152,10 +151,21 @@
                     </li>
                     <li>
                         <a target="_self" class="inner-link" href="#press">Press</a>
+                    </li>-->
+                    <li>
+                        <a target="_self" href="#topics">Topics</a>
+                    </li>
+                    
+                    <li>
+                        <a target="_self" href="#about">About</a>
```

Comment:
This is unnecessary.

Reply:
what is? The white space?

Reply:
or the about link?

Reply:
About link, it is unnecessary for navbar